<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ex1Class</class>
 <widget class="QMainWindow" name="ex1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ex1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.PingFang SC</family>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.PingFang SC'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.PingFang SC'; font-size:14pt;&quot;&gt;简易学生信息管理系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>701</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.PingFang SC</family>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.PingFang SC'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.PingFang SC';&quot;&gt;输出&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_used_time">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>耗时114514MS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filename">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>61</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_file">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>63</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search_file">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>94</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索学生</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search_single">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>94</y>
      <width>104</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询该生平均成绩</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_name_or_num">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>91</y>
      <width>256</width>
      <height>29</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>姓名/学号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search_all">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>94</y>
      <width>104</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询全体平均成绩</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_class">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>54</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择班级</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_class">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>51</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>班级</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ex1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_file</sender>
   <signal>clicked()</signal>
   <receiver>ex1Class</receiver>
   <slot>fileOpenAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>fileMenu()</signal>
  <slot>fileOpenAction()</slot>
 </slots>
</ui>
